### PR TITLE
From definition to principles.

### PR DESCRIPTION
**Pull request by CoralineAda**
New revision to the Ethical Source Principles (formerly "Definition") based on working group collaboration.

[Preview](https://deploy-preview-89--ethicalsource.netlify.app/)

### CODE REVIEW

**Comment by CoralineAda** [comment]
Tagging in @willbarton

**Comment by CoralineAda** [comment]
See the site preview by clicking on the 'details' link next to 'deploy/netlift' under the 'All checks have passed' heading.

**Comment by UncannyBingo** [comment]
Can we add a special badge for projects that are doing all the things? A special glowy, shiny badge :D

**Review by willbarton — APPROVED** [review]
Looks good to me!

The only question I have is, should we keep the translations of the older definition? This question also makes me wonder if we should start versioning this document so that the relationship between translations is clear.

File: content/definition.md [hunk 1]
```diff
@@ -1,27 +1,40 @@
...
+3. **We strive to make our community welcoming and just.** We have clear and documented rules for governance, and we consistently and fairly enforce a comprehensive code of conduct.
+4. **Our work puts accessibility first.** We design our software to be accessible, ensuring that all features and functionality are available to all users, including users who may rely on assistive devices.
+5. **Our work prioritizes user safety.** We design our software with features and safeguards to minimize the risk of abuse or harm to others through the its use or misuse.
+6. **Our work protects user privacy.** We design our software with controls for the complete deletion of any personal data gathered from its use, and for exporting any such data to the end-user on request in a human-readable or interoperable and open format.
+7. **Our work deserves to be supported.** We are empowered to request remuneration in the form of code contributions, financial contributions, or other forms of voluntary support from organizations that derive commercial value from our software.
```
Comment (tobie) [inline content/definition.md]: I'm uneasy with this framing which implies power dynamics in the relationship that favor users of the commons in opposition to those giving away their work.

Not sure how to frame it, but maybe taking a step back and expecting that everyone deserves dignity and proper pay, or something like that?

File: content/definition.md [hunk 2]
```diff
@@ -1,27 +1,40 @@
...
-5. **It prioritizes user safety.** The software must be designed with features and safeguards that minimize the risk of abuse or harm to others through the use of the software.
-6. **It protects user privacy.** If the operation of the software collects end-user data, the software must be designed with provisions for its operators to completely delete or provide to the end-user any and all data generated and stored from the end-user’s usage of the software. The export mechanism must provide the data in a human-readable or widely interoperable open format.
-7. **It encourages fair compensation.** The software project’s maintainers may, at their discretion, request remuneration in the form of code contributions, financial consideration, or other forms of voluntary support from organizations that derive commercial value from the software.
+1. **Our work benefits the commons.** We release our software under conditions that do not prohibit modification, derivative works, and similar use and reuse in the general spirit of free, open source, source available, or other commons distribution.
```
Comment (tobie) [inline content/definition.md]: Do we want to limit this definition to software or open it up more broadly? "contributions," "digital commons," "work," all come to mind.

File: content/definition.md [hunk 3]
```diff
@@ -1,27 +1,40 @@
...
+2. **Our work is done in the open.** We make our source code publicly available, develop and maintain it in public view, and welcome  public contributions.
+3. **We strive to make our community welcoming and just.** We have clear and documented rules for governance, and we consistently and fairly enforce a comprehensive code of conduct.
+4. **Our work puts accessibility first.** We design our software to be accessible, ensuring that all features and functionality are available to all users, including users who may rely on assistive devices.
+5. **Our work prioritizes user safety.** We design our software with features and safeguards to minimize the risk of abuse or harm to others through the its use or misuse.
```
Comment (tobie) [inline content/definition.md]: s/users/people/?

```suggestion
5. **Our work prioritizes the safety of people.** We design our software with features and safeguards to minimize the risk of abuse or harm to others through the its use or misuse.
```

File: content/definition.md [hunk 4]
```diff
@@ -1,27 +1,40 @@
...
+6. **Our work protects user privacy.** We design our software with controls for the complete deletion of any personal data gathered from its use, and for exporting any such data to the end-user on request in a human-readable or interoperable and open format.
```
Comment (tobie) [inline content/definition.md]: s/users/people/?

Also requesting both human readable and interoperable format, not one or the other.

```suggestion
6. **Our work protects privacy.** We design our software with controls for the complete deletion of any personal data gathered from its use, and for exporting any such data to people requesting it in both a human-readable format and in an interoperable and open format.
```

File: content/definition.md [hunk 5]
```diff
@@ -1,27 +1,40 @@
...
+7. **Our work deserves to be supported.** We are empowered to request remuneration in the form of code contributions, financial contributions, or other forms of voluntary support from organizations that derive commercial value from our software.
 
-A community that practices **Ethical Open Source** reflects its alignment with these principles through the actions and aspirations of its members.
+We measure our success in adhering to these principles on _impact_ rather than _intent_.
```
Comment (tobie) [inline content/definition.md]: 🔥🔥🔥

File: content/definition.md [hunk 6]
```diff
@@ -1,27 +1,40 @@
...
 
-[https://img.shields.io/badge/open-ethical-%234baaaa](https://img.shields.io/badge/open-ethical-%234baaaa)
 
 
 ## Attribution and Contributions
-Maintaining this definition is the collaborative work of the [Ethical Source Working Group](/apply). Pull requests and issues in our [source code repository](https://github.com/ethicalSource/ethicalsource.dev) are encouraged. Special thanks are extended to [Nate Berkopec](https://nateberkopec.com), [Will Barton](https://github.com/willbarton/), [Steve Phillips](https://tryingtobeawesome.com), and [Madalyn Parker](https://twitter.com/madalynrose) for their contributions.
+Maintaining the Ethical Source Principles is the collaborative work of the [Ethical Source Working Group](/apply). Pull requests and issues in our [source code repository](https://github.com/ethicalSource/ethicalsource.dev) are encouraged. Special thanks are extended to [Nate Berkopec](https://nateberkopec.com), [Will Barton](https://github.com/willbarton/), [Steve Phillips](https://tryingtobeawesome.com), and [Madalyn Parker](https://twitter.com/madalynrose) for their contributions.
```
Comment (tobie) [inline content/definition.md]:
```suggestion
Maintaining the Ethical Source Principles is the collaborative work of the [Organization for Ethical Source](/apply). Pull requests and issues in our [source code repository](https://github.com/ethicalSource/ethicalsource.dev) are encouraged. Special thanks are extended to [Nate Berkopec](https://nateberkopec.com), [Will Barton](https://github.com/willbarton/), [Steve Phillips](https://tryingtobeawesome.com), and [Madalyn Parker](https://twitter.com/madalynrose) for their contributions.
```

File: content/definition.md [hunk 7]
```diff
@@ -1,27 +1,40 @@
 +++
```
Comment (tobie) [inline content/definition.md]: Shouldn't this be a redirect instead of a copy of /principles ?

File: content/definition.md (hunk 2 above)
Comment (tobie) [inline content/definition.md]:
```suggestion
1. **Our work benefits the commons.** We release our software under conditions that allow modification, derivative works, and similar use and reuse in the general spirit of free, open source, source available, or other commons distribution.
```

Or "promote."

Comment (tobie) [reply content/definition.md]: Do we need to note that it's ok to limit the reuse within a given community or something of that nature? Not super sure how to phrase that.

File: content/definition.md [hunk 8]
```diff
@@ -1,27 +1,40 @@
...
-7. **It encourages fair compensation.** The software project’s maintainers may, at their discretion, request remuneration in the form of code contributions, financial consideration, or other forms of voluntary support from organizations that derive commercial value from the software.
+1. **Our work benefits the commons.** We release our software under conditions that do not prohibit modification, derivative works, and similar use and reuse in the general spirit of free, open source, source available, or other commons distribution.
+2. **Our work is done in the open.** We make our source code publicly available, develop and maintain it in public view, and welcome  public contributions.
+3. **We strive to make our community welcoming and just.** We have clear and documented rules for governance, and we consistently and fairly enforce a comprehensive code of conduct.
```
Comment (tobie) [inline content/definition.md]: I'd be inclined to add something here about favoring autonomy of contributors, consensus decision-making, etc. But might be too specific for now.

**Review by tobie — COMMENTED** [review]
Add a series of comments some rather straightforward, others inviting a broader reflection. It can be OK to file the broader ones separately and move forward with landing this first.

**Pull request merged**